packages/drivers/local-driver/src/localResolver.ts

```diff
@@ -68,11 +68,12 @@ export class LocalResolver implements IUrlResolver {
         const fluidResolvedUrl = resolvedUrl as IFluidResolvedUrl;
 
         const parsedUrl = parse(fluidResolvedUrl.url);
-        if (parsedUrl.pathname === undefined) {
+        // eslint-disable-next-line no-null/no-null
+        if (parsedUrl.pathname === null) {
```

Comment:
Updated types caught a bug: 'pathname' is never undefined, but can be null.  (Verified in console.)